Common/AssemblyResolver.cs

```diff
@@ -1,6 +1,6 @@
 /*
  * SonarQube Roslyn SDK
- * Copyright (C) 2015-2017 SonarSource SA
+ * Copyright (C) 2015-2022 SonarSource SA
```

Comment:
This file needs to be updated too: 
https://github.com/SonarSource/sonarqube-roslyn-sdk/blob/f64ba6a8d91aaedb000516f09c8395cfdc83884d/NOTICE.txt#L2